benchmarks/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "benchmarks"
+version = "0.1.0"
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+exclude.workspace = true
+
+[[bin]]
+name = "benchmarks"
```

Comment:
I would use `miden-tx-bench` (the default) or maybe just `tx-bench` as the executable name.